code/api/src/modules/user/resolvers.js

```diff
@@ -77,3 +77,23 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
+// Add user style
+export async function addStyle(parentValue, { id, survey_results } ) {
+  var array = survey_results.split(", ")
+  var counts = array.reduce((a, c) => {
+    a[c] = (a[c] || 0) + 1;
```

Comment:
I had no idea that it was possible to memoize within a statement like this.  That's really good to know!